deliver/lib/deliver/upload_metadata.rb

```diff
@@ -226,30 +226,47 @@ def verify_available_languages!(options)
       # Collect all languages we need
       # We only care about languages from user provided values
       # as the other languages are on iTC already anyway
-      v = options[:app].edit_version
-      UI.user_error!("Could not find a version to edit for app '#{options[:app].name}', the app metadata is read-only currently") unless v
+      v = options[:app].details
```

Comment:
why the switch from `.edit_version` to `.details` here?

Reply:
I think I looked at what the ITC web interface did when you added a new language, and the request seemed to correspond with what .details was doing, rather than .edit_version .. maybe. sorry my memory is only reliable up to 7 days max